src/base-styles/helperClasses/border/focus.css

```diff
@@ -0,0 +1,24 @@
+/*---
+title: Focus Style
+name: focusStyle
+section: Border Helpers
+---
+
+Adds standard focus style to elements
+
+class                    | description
+------------------------ | -------------------------------------------------------
+`border--focus`          | Adds standard box shadow focus style
+`border--focus--notransition`  | Adds standard box shadow focus style without transition
+*/
+
+.border--focus:focus {
+  outline: 0;
+  box-shadow: 0 0 0 2px #B7D7E5;
+  transition: box-shadow 250ms var(--motion-easing-default);
+}
+
+.border--focus--notransition:focus {
```

Comment:
tiniest nitpick ever - would be good to make the variant camelCase to match other classes: `border--focus--noTransition`